src/main/java/uk/gov/cabinetoffice/csl/util/Utils.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.cabinetoffice.csl.util;
+
+public class Utils {
```

Comment:
Could we make this into a utils spring component maybe? Just to avoid using static methods

Reply:
Done